client/Components/Main.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Home from './Home.jsx';
+import Menu from './Hearder/Menu.jsx';
+import Footer from './Footer/Footer.jsx';
+
+import './../js/recipes.js';
+import '../Style/test.scss';

```

Comment:
What does this SCSS test?

Reply:
I was using it to test my app. I will remain it.